### ISSUE TITLE
collision between `hl-em` tag `{-` and blade's comment tag `{{--`

### ISSUE THREAD

**Issue by brendt** [tempestphp/highlight#10]

**Comment by forxer** [comment]
Hello @brendt, this problem is still present in 2.3.1

```blade
{{-- Blade comment --}}
```

produce:

```html
<span class="hl-comment">{<span class="hl-deletion">- Blade comment -</span>}</span>
```

**Comment by brendt** [comment]
Hm, thanks for letting me know, that's probably a regression and I'll look into it!

**Comment by brendt** [comment]
Fixed in 2.3.3 https://github.com/tempestphp/highlight/releases/tag/2.3.3
